npm/README.md

```diff
@@ -0,0 +1,114 @@
+# electron-prebuilt
+
+[![Travis build status](http://img.shields.io/travis/electron-userland/electron-prebuilt.svg?style=flat)](http://travis-ci.org/electron-userland/electron-prebuilt)
```

Comment:
> use the one from root?

I like the idea of npm.im/electron having the same readme as ghub.io/electron, as that is what users normally expect. But if we do that, I think we'll need to preserve some parts of the prebuilt readme, as it has useful installation and troubleshooting info that's not present in the top-level readme.